File: Assets/Scripts/PlayerCasting.js
static var distanceFromTarget : float;

var toTarget : float;

function Update() {
		var hit : RaycastHit;
		if (Physics.Raycast (transform.position, transform.TransformDirection(Vector3.forward), hit)) {
			toTarget = hit.distance;
			distanceFromTarget = toTarget;
	}
}

File: Assets/Scripts/PlayerCasting.cs
﻿using System.Collections;
using System.Collections.Generic;
using UnityEngine;
public class PlayerCasting : MonoBehaviour
{
	public static float distanceFromTarget;
	float toTarget;

	// Use this for initialization
	void Start()
	{

	}

	// Update is called once per frame
	void Update()
	{

		RaycastHit hit;
		if (Physics.Raycast(transform.position, transform.TransformDirection(Vector3.forward), out hit))
		{
			toTarget = hit.distance;
			distanceFromTarget = toTarget;
		}
	}
}